<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgImageScale</class>
 <widget class="QWidget" name="WdgImageScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scale To New Size</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="MultiLockButton" name="lock_pixel" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupPixelSize">
     <property name="title">
      <string>Size in Pixels</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <spacer name="spacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelWidth">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>intPixelWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="intPixelWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5" rowspan="2">
       <widget class="KoAspectButton" name="aspectPixels">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="6" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelHeight">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>intPixelHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="intPixelHeight">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cmbWidthPixelUnit">
        <item>
         <property name="text">
          <string>Pixels</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Percent (%)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cmbHeightPixelUnit">
        <item>
         <property name="text">
          <string>Pixels</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Percent (%)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doublePercentageWidth">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doublePercentageHeight">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelFilter">
        <property name="text">
         <string>Interpolation:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>doublePhysicalWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="KisCmbIDList" name="cmbFilterType" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="MultiLockButton" name="lock_print" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupPhysicalsize">
     <property name="title">
      <string>Print Size</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <spacer name="spacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doublePhysicalWidth">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="cmbWidthUnit"/>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="KoAspectButton" name="aspectPhysical">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="spacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="cmbHeightUnit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPhysicalWidth">
        <property name="text">
         <string>Print Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>doublePhysicalWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPhysicalHeight">
        <property name="text">
         <string>Print Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>doublePhysicalHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doublePhysicalHeight">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="MultiLockButton" name="lock_resolution" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Resolution</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="spacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>181</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelResolution">
        <property name="text">
         <string>Resolution:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleResolution">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelResolutionUnit">
        <property name="text">
         <string>pixels/inch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends></extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QLabel</extends>
   <header>KoAspectButton.h</header>
  </customwidget>
  <customwidget>
   <class>MultiLockButton</class>
   <extends>QWidget</extends>
   <header>multilock_button.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>intPixelWidth</tabstop>
  <tabstop>doublePercentageWidth</tabstop>
  <tabstop>intPixelHeight</tabstop>
  <tabstop>doublePercentageHeight</tabstop>
  <tabstop>cmbWidthPixelUnit</tabstop>
  <tabstop>cmbHeightPixelUnit</tabstop>
  <tabstop>doublePhysicalWidth</tabstop>
  <tabstop>doublePhysicalHeight</tabstop>
  <tabstop>cmbWidthUnit</tabstop>
  <tabstop>cmbHeightUnit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
